### ISSUE TITLE
Post Checkout: improve color contrast on receipts for readability

### ISSUE THREAD

**Issue by mriyazuddin** [Automattic/wp-calypso#40638]
Request to update the color contrast on receipts. The text in light grey isn't suitable for printing or reading. Suggestion to make that black in color so its more visible and easy to read. Here's a screenshot of a receipt with light colors indicated:

![Annotation on 2020-04-04 at 23-18-42](https://user-images.githubusercontent.com/66797/78457949-cdcf2a00-7662-11ea-985b-1c25bd3a100b.png)

#### Context / Source

User feedback: 2836622-zen

**Comment by fditrapani** [comment]
Some updates were recently made to the receipt that helped with contrast:

![Image](https://github.com/user-attachments/assets/2ff214d5-b64a-407d-bd10-2e6b8e9d4b1b)
